### ISSUE TITLE
RxNetty and 2.x vs. old-2.x 

### ISSUE THREAD

**Issue by nrdavid** [Netflix/zuul#121]
I had been tracking the RxNetty based version of Zuul that was recently moved to the "old-2.x" branch. Is RxNetty no longer on the roadmap? Is the intention for Zuul 2 to provide some sort of non-blocking semantics - either through Netty/RxNetty or the Servlet 3.1 API?

Thank you

**Comment by kerumai** [comment]
Hey David. An RxNetty-based Zuul is still on the roadmap. I moved the 2.x branch as that was an early prototype, and we needed to go back and change some of the interfaces, so wanted to flag that that branch was ending.
I've been working on the "zuul2-bridge" branch, and will at some point soon, use that as the new 2.x, although we'll probably just be building snapshots for a while, until sure that the interfaces have settled down.

One of the intents of the zuul2-bridge branch was to separate out the zuul-core module from being dependent on netty or servlets, and then make the filter interfaces and FilterProcessor the same for operating on either RxNetty or blocking Servlets.

thanks
